islands/components/DollProgress.tsx

```diff
@@ -0,0 +1,44 @@
+import type { MiniMe } from "../../loaders/MiniMe/minime.ts";
+import { ProductDetailsPage } from "apps/commerce/types.ts";
+import { useState } from "preact/hooks";
+
+/**@title Informações da Mini Me*/
+interface Props {
+  dollParts: MiniMe;
+
+  /**@title Título da Mini Me*/
+  title: string;
+
+  page: ProductDetailsPage | null;
+}
+
+export default function DollTitle(props: Props) {
+    const [stepCount, setStepCount] = useState(Number(localStorage.getItem('step')));
+    const [stepSelectedCount, setStepSelectedCount] = useState(localStorage.getItem('selectedStep') || "pele");
+
+  return (
+    <>
+      <div class="flex items-center relative mb-[36px] w-full">
+        {Object.keys(props.dollParts).map((i, index) => (
```

Comment:
Variáveis não utilizadas podem ser nomeadas com `_`.
```suggestion
        {Object.keys(props.dollParts).map((_, index) => (
```